src/Core/src/PublicAPI/net-ios/PublicAPI.Shipped.txt

```diff
@@ -2795,7 +2795,6 @@ static Microsoft.Maui.Platform.ImageViewExtensions.Clear(this UIKit.UIImageView!
 static Microsoft.Maui.Platform.ImageViewExtensions.UpdateAspect(this UIKit.UIImageView! imageView, Microsoft.Maui.IImage! image) -> void
 static Microsoft.Maui.Platform.ImageViewExtensions.UpdateIsAnimationPlaying(this UIKit.UIImageView! imageView, Microsoft.Maui.IImageSourcePart! image) -> void
 static Microsoft.Maui.Platform.ImageViewExtensions.UpdateSource(this UIKit.UIImageView! imageView, UIKit.UIImage? uIImage, Microsoft.Maui.IImageSourcePart! image) -> void
-static Microsoft.Maui.Platform.ImageViewExtensions.UpdateSourceAsync(this UIKit.UIImageView! imageView, Microsoft.Maui.IImageSourcePart! image, Microsoft.Maui.IImageSourceServiceProvider! services, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<Microsoft.Maui.IImageSourceServiceResult<UIKit.UIImage!>?>!
```

Comment:
We cannot modify the shipped APIs - a tool does that.

Also, this means a breaking change so we need to avoid that at all costs at this point.